Add sync/force sync behavior to monitors/SLOs/Dashboards docs (#2199)
DataDog/datadog-operator: docs/datadog_slo.md

    ```

    This automatically creates a new SLO in Datadog. You can find it on the [SLOs][8] page of your Datadog account.

>     Datadog Operator will occasionally reconcile and keep SLOs in line with the given configuration. There is also a force 
>     sync every hour, so if a SLO is deleted in the Datadog UI, it will be back up in under an hour.

Reviewer: ```suggestion
    Datadog Operator occasionally reconciles and keeps SLOs in line with the given configuration. There is also a force 
    sync every hour, so if a user deletes an SLO in the Datadog UI, Datadog Operator restores it in under an hour.
```
